### ISSUE TITLE
when importing OpenAPI fails

### ISSUE THREAD

**Issue by jeremychoi** [zaproxy/zaproxy#7386]
### Is your feature request related to a problem? Please describe.

OpenAPI Support version: 27.0.0

# TL;DR; 
1. Is it possible to show(logging) a line number of an OpenAPI file when an error occurs?
2. Is it possible to continue importing the rest if only a certain parameter causes an error, rather than failing the import function completely?

# Details

When an error occurs during importing OpenAPI files, currently it results in failing the import action completely. The problem is it doesn't show the location of the openAPI file that causes the error, so It is hard to debug. Hence, TL;DR; `#1`. Additionally, it would be useful if importing could continue with the rest of the endpoints that are irrelevant with a certain parameter that causes an error like the following.  

A blank(also, with space) value(of 'name') in the openAPI file like the following will cause an java.lang.NullPointerException error , which causes to fail to import an OpenAPI file:
```
{
  "openapi": "3.0.1",
  "info": {
    "version": "1.0",
    "title": "null-pointer-exception reproducer"
  },
  "servers": [
    {
      "url": "/"
    }
  ],
  "paths": {
    "/testapi": {
      "post": {
        "summary": "Create",
        "parameters": [
          {
            "name": "access_token",
            "in": "query",
            "description": "A personal Access Token",
            "required": true,
            "schema": {
              "type": "string"
            }
          },
          {
            "name": "system_name",
            "in": "query",
            "description": "System Name",
            "schema": {
              "type": "string"
            }
          },
          {
            "name": "",
            "in": "query",
            "description": "Extra parameters",
            "style": "form",
            "explode": false,
            "schema": {
              "type": "array",
              "items": {
                "type": "object"
              }
            }
          }
        ],
        "responses": {
          "default": {
            "description": "success",
            "content": {}
          }
        }
      }
    }
  },
  "components": {}
}
```

The error details are as follows:

```
java.lang.NullPointerException
	at org.zaproxy.zap.extension.formhandler.FormHandlerValueGenerator.getValue(FormHandlerValueGenerator.java:50)
	at org.zaproxy.zap.extension.openapi.generators.ValueGenerator.getValue(ValueGenerator.java:54)
	at org.zaproxy.zap.extension.openapi.generators.DataGenerator.generateValue(DataGenerator.java:197)
	at org.zaproxy.zap.extension.openapi.generators.DataGenerator.generateArrayValue(DataGenerator.java:153)
	at org.zaproxy.zap.extension.openapi.generators.DataGenerator.generateParam(DataGenerator.java:136)
	at org.zaproxy.zap.extension.openapi.generators.DataGenerator.generate(DataGenerator.java:75)
	at org.zaproxy.zap.extension.openapi.generators.PathGenerator.generateFullPath(PathGenerator.java:42)
	at org.zaproxy.zap.extension.openapi.converter.swagger.RequestModelConverter.generatePath(RequestModelConverter.java:60)
	at org.zaproxy.zap.extension.openapi.converter.swagger.RequestModelConverter.convert(RequestModelConverter.java:46)
	at org.zaproxy.zap.extension.openapi.converter.swagger.SwaggerConverter.convertToRequest(SwaggerConverter.java:192)
	at org.zaproxy.zap.extension.openapi.converter.swagger.SwaggerConverter.getRequestModels(SwaggerConverter.java:186)
	at org.zaproxy.zap.extension.openapi.ExtensionOpenApi$1.run(ExtensionOpenApi.java:378)
Failed to parse OpenAPI definition.



```

### Describe the solution you'd like

Is it possible to show(logging) a line number of an OpenAPI file when an error occurs?
Is it possible to continue importing the rest if only a certain parameter causes an error, rather than failing the import function completely?

### Describe alternatives you've considered

the same as the solution above

### Would you like to help fix this issue?

- [v] Yes

**Comment by thc202** [comment]
What versions are being used?
